### PR TITLE
Initialized CONTRIBUTING instructions

### PR DESCRIPTION
**Pull request by pranjalpokharel7**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+#### Build Instructions
+
+These are general guidelines you can follow to keep the build 
+process consistent across all systems.
+
+Create build.bat (Windows) and build.sh (Linux) that contains the 
+build instructions for your program depending upon the system. The
+build file should be placed in the path - 
+`./Samples/<your-sample-project-directory>/bin/<build-script>`
```
Comment (bhattaraiashish) [inline CONTRIBUTING.md]: `./Samples/<your-sample-project-directory>/`
just there, the directory may look like:
main.c   build.bat   build.sh [bin]

**Review by bhattaraiashish — COMMENTED** [review]
please check the comment in the file and make the change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ process consistent across all systems.
 Create build.bat (Windows) and build.sh (Linux) that contains the 
 build instructions for your program depending upon the system. The
 build file should be placed in the path - 
-`./Samples/<your-sample-project-directory>/bin/<build-script>`
+`./Samples/<your-sample-project-directory>/<build-scripts>`
```
Comment (pranjalpokharel7) [inline CONTRIBUTING.md:59]: I think this should make things clearer on the user side.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,6 +66,16 @@ creating a build file :
 - The output of the files should be placed in directory `bin` relative to the local project directory.
 - All the bin directories (including object files) are to be ignored in the repo.
 
+#### Final Directory Structure
```
Comment (pranjalpokharel7) [inline CONTRIBUTING.md:69]: I've also added a final directory structure, should be useful for someone who is knowledgeable working with C projects and wants to skip to the end.

**Pull request merged**